org.eclipse.xtext.maven.plugin/README.txt

```diff
@@ -0,0 +1,56 @@
+To run the integration tests, the `target/.m2test` folder must first be populated with Maven dependencies and with the IT-SNAPSHOT jar of this Maven plugin.
```

Comment:
should this be a md file?

Reply:
Yes, I'll transform that

Reply:
Fixed in https://github.com/xtext/xtext-monorepo/pull/17/commits/4cd80eed85a3d758d9323edb2ec7171b6774f8d8